bleachbit/FileUtilities.py

```diff
@@ -237,13 +237,12 @@ def write(parser, ini_file):
         removing a cast to str. This is needed to handle unicode chars.
         """
         if parser._defaults:
-            ini_file.write("[%s]\n" % "DEFAULT")
+            ini_file.write("[DEFAULT]\n")
```

Comment:
I'm just leaving a note here to prevent possible confusion.

The reason I left out the `f` in this statement was because it's formatting from a literal string. Thus it's better to inline it into the string it's formatting.